### PR TITLE
[FDD, FT0] Use of simple check for vertex event and proper logic for FT0 Central and semiCentral Trigger

### PR DESCRIPTION
**Pull request by arvindkhuntia**
@jotwinow @andreasmolander

### CODE REVIEW

**Comment by arvindkhuntia** [comment]
Hi @afurs, I have just changes the logic for C and SC trigger for FT0.

File: Modules/FIT/FT0/src/DigitQcTask.cxx [hunk 1]
```diff
@@ -628,9 +628,9 @@ void DigitQcTask::monitorData(o2::framework::ProcessingContext& ctx)
     switch (mTrgModeSide) {
       case TrgModeSide::kAplusC:
         if (mTrgModeThresholdVar == TrgModeThresholdVar::kAmpl) {
-          if (pmSumAmplA + pmSumAmplC >= 2 * mTrgThresholdCenA)
+          if (pmSumAmplA + pmSumAmplC > 2 * mTrgThresholdCenA)
             mMapTrgSoftware[o2::ft0::Triggers::bitCen] = true;
-          else if (pmSumAmplA + pmSumAmplC >= 2 * mTrgThresholdSCenA)
+          else if (pmSumAmplA + pmSumAmplC > 2 * mTrgThresholdSCenA && !mMapTrgSoftware[o2::ft0::Triggers::bitCen])
```
Comment (afurs) [inline Modules/FIT/FT0/src/DigitQcTask.cxx:633]: @arvindkhuntia why did you add `... && !mMapTrgSoftware[o2::ft0::Triggers::bitCen]` ? there are already `if-else` construction, where first `if` checks `bitCen`

Comment (arvindkhuntia) [reply Modules/FIT/FT0/src/DigitQcTask.cxx]: Hi @afurs, you are right, just removed it.

**Review by afurs — APPROVED** [review]

**Review by andreasmolander — APPROVED** [review]

**Review by knopers8 — APPROVED** [review]
